mkosi/__init__.py

```diff
@@ -298,17 +298,20 @@ def can_minimize(self) -> bool:
 
 
 class Distribution(enum.Enum):
-    fedora = 1
-    debian = 2
-    ubuntu = 3
-    arch = 4
-    opensuse = 5
-    mageia = 6
-    centos = 7
-    centos_epel = 8
-    clear = 9
-    photon = 10
-    openmandriva = 11
+    fedora = "fedora"
+    debian = "debian"
+    ubuntu = "ubuntu"
+    arch = "arch"
+    opensuse = "opensuse"
+    mageia = "mageia"
+    centos = "centos"
+    centos_epel = "centos_epel"
+    clear = "clear"
+    photon = "photon"
+    openmandriva = "openmandriva"
+
+    def __str__(self) -> str:
+        return cast(str, self.value)
```

Comment:
I don't think this renaming is necessary. Couldn't the `__str__` method could be `cast(str, self.name)` instead? This way the members don't need to repeat the keys. Easiest would be to use `enum.auto()` anyways.